scanner/updater/export.go

```diff
@@ -117,7 +117,8 @@ func Export(ctx context.Context, outputDir string, opts *ExportOptions) error {
 }
 
 func manualOpts(ctx context.Context) ([]updates.ManagerOption, error) {
-	manualSet, err := manual.UpdaterSet(ctx)
+	uri := "https://raw.githubusercontent.com/stackrox/stackrox/9ad0817d2a437f59ee4337f4eb72c024a8180a34/scanner/updater/manual/vulns.json"
```

Comment:
This should point to some reference that is updated automatically if changed in the repository. With a commit SHA, you will get the same content regardless of the changes you make in the file. Is that your intention?

Reply:
This URI is just for testing purpose, the default is `DefaultURL = "https://raw.githubusercontent.com/stackrox/stackrox/master/scanner/updater/manual/vulns.json"`
I will remove the testing uri before merge